Formula/a/autobrr.rb

```diff
@@ -0,0 +1,46 @@
+class Autobrr < Formula
+  desc "Modern, easy to use download automation for torrents and usenet"
+  homepage "https://autobrr.com/"
+  url "https://github.com/autobrr/autobrr/archive/refs/tags/v1.36.0.tar.gz"
+  sha256 "631ee5f858c5952e1d7a8c27b13fe33aea9021ffd814bb540f0e494291168309"
+  license "GPL-2.0-or-later"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    system "npx", "pnpm", "install", "--dir", "web"
+    system "npx", "pnpm", "--dir", "web", "run", "build"
+
+    ldflags = "-s -w -X main.version=#{version} -X main.commit=#{tap.user}"
+
+    system "go", "build", *std_go_args(output: bin/"autobrr", ldflags: ldflags), "./cmd/autobrr"
+    system "go", "build", *std_go_args(output: bin/"autobrrctl", ldflags: ldflags), "./cmd/autobrrctl"
+  end
+
+  def post_install
+    (var/"autobrr").mkpath
+  end
+
+  service do
+    run [opt_bin/"autobrr", "--config", var/"autobrr/"]
+    keep_alive true
+    log_path var/"log/autobrr.log"
+  end
+
+  test do
+    pid = fork do
+      exec "#{bin}/autobrr", "--config", testpath
+    end
+    sleep 2
+
+    begin
+      system "curl", "-s", "-o", "/dev/null", "-w", "%{http_code}", "http://127.0.0.1:7474/api/healthz/liveness", "|", "grep", "200"
```

Comment:
Why not let curl fail on non-success statuses?

Reply:
Fixed